lib/apps/customer_app/features/auth/ui/screens/signin_screen.dart

```diff
@@ -91,6 +92,26 @@ class SigninScreen extends ConsumerWidget {
                                     );
                                   },
                                 ),
+                                const SizedBox(
+                                  height: 10,
+                                ),
+                                 Row(
+                                  children: [
+                                    const Text("Forgot Password?",style: TextStyle(
+                                      color: Color(0xff949494),
+                                      fontSize: 14.0,
+                                      fontWeight: FontWeight.w600,
+                                    ), ),
+                                   const SizedBox(
+                                      width: 3,
+                                    ),
+                                    InkWell(
+                                      onTap: () => Navigator.of(context)
+                                          .pushReplacementNamed(AuthRoutes.forgetPassword),
+                                      child: const Text('Reset it'),
```

Comment:
using pushReplacementNamed means the user can't go back to the login page as you've cleared it from the route.
The 2 options here are:

-  Change it to pushNamed so that login still stays in the router stack or
- Add a link in the forget password page that can route you back to the login page using pushReplacementNamed again.